Mathlib/Order/InitialSeg.lean

```diff
@@ -35,22 +39,15 @@ These notations belong to the `InitialSeg` locale.
 * `α <i β` is an abbreviation for `(· < ·) ≺i (· < ·)`.
 -/
 
-
-/-!
-### Initial segments
-
-Order embeddings whose range is an initial segment of `s` (i.e., if `b` belongs to the range, then
-any `b' < b` also belongs to the range). The type of these embeddings from `r` to `s` is called
-`InitialSeg r s`, and denoted by `r ≼i s`.
--/
+/-! ### Initial segment embeddings -/
 
 variable {α : Type*} {β : Type*} {γ : Type*} {r : α → α → Prop} {s : β → β → Prop}
   {t : γ → γ → Prop}
 
 open Function
 
 /-- If `r` is a relation on `α` and `s` in a relation on `β`, then `f : r ≼i s` is an order
-embedding whose range is an initial segment. That is, whenever `b < f a` in `β` then `b` is in the
+embedding whose `Set.range` is a lower set. That is, whenever `b < f a` in `β` then `b` is in the
```

Comment:
Didn't we discuss changing the definition to make this literally true? Or am I misremembering the conclusion

Reply:
This can't be done as `IsLowerSet` is defined in terms of an order relation, while `InitialSeg` is stated in terms of an arbitrary relation.

However, we do already have `InitialSeg.isLowerSet_range` for initial segments between `<` in a preorder and `<` in a partial order.